src/helm.exec.ts

```diff
@@ -173,6 +173,26 @@ export function helmGet(resourceNode: explorer.ResourceNode) {
     });
 }
 
+// helmPackage runs the Helm package on a chart within your project.
+export function helmPackage() {
+    pickChart((path) => {
+        vscode.window.showInputBox({
```

Comment:
This might be better done with `showOpenDialog`?  Then the user can enter a path manually or select one using the GUI.

Reply:
For me as a user opening a dialog would be a better option!
After taking a quick look about `showOpenDialog` and after testing it, it supports only selection through GUI. Will it be enough for you as a user or you need the ability to enter the path manually ? Also will we have to use both `showOpenDialog` and `showInputBox` in this case or this is a better way to do it ?

Reply:
I'm sorry, I explained myself badly.  As you rightly say, `showOpenDialog` always shows a GUI.  But if the user really wants to type or paste a path then they can do that in the text box on the dialog (at least on Windows).  So although it is not quite as seamless as `showInputBox` in this case, it still suffices, and is easier for what I think is the common case where the user wants to browse.  So no, we don't need to also use `showInputBox` - `showOpenDialog` should serve both audiences.  Hope that makes more sense now!